horace_core/sqw/@DnDBase/get_proj_and_pbin.m

```diff
@@ -0,0 +1,36 @@
+function [proj, pbin] = get_proj_and_pbin(w)
+% Retrieve the projection and binning of an sqw or dnd object
```

Comment:
again checking why this is a dndbase method not sqwdndbase

Reply:
as above -- works on image.